x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/ingest_pipeline/install.ts

```diff
@@ -157,13 +183,13 @@ const getNameAndExtension = (
 export const getPipelineNameForInstallation = ({
   pipelineName,
   dataset,
-  datasourceName,
   packageName,
+  packageVersion,
 }: {
   pipelineName: string;
   dataset: Dataset;
-  datasourceName: string;
   packageName: string;
+  packageVersion: string;
 }): string => {
-  return `${dataset.type}-${packageName}-${datasourceName}-${dataset.name}-${pipelineName}`;
+  return `${dataset.type}-${dataset.name}-${packageVersion}-${pipelineName}`;
```

Comment:
Will all pipelines be postfixed by the pipelineName? I think the pipeline where the data lands at should be `${dataset.type}-${dataset.name}-${packageVersion}` and only the additional pipelines need the postfix. This will make it very easy to reference the pipeline in the index template as it always uses the same convention.